tex/gregoriotex.lua

```diff
@@ -214,7 +214,8 @@ local function init(arg, enable_height_computation)
   if enable_height_computation then
     new_line_heights = {}
 
-    local mcb_version = luatexbase.get_module_version('luatexbase-mcb')
+    local mcb_version = luatexbase.get_module_version and
+        luatexbase.get_module_version('luatexbase-mcb') or 9999
```

Comment:
ok for me, this use of `and` and `or` in Lua is really nice, too bad it's not that common in other languages (though it works in most).